#Expyriment 0.11.dev5 (Python 3.11.9), .xpe-file, coding: cp1252
#date: Wed Jan 22 2025 19:25:16
#sha1: d5019a
#modules: 
#display: size=(800, 600), window_mode=False, opengl=2
#os: uname_result(system='Windows', node='Geoff3', release='10', version='10.0.22631', machine='AMD64')
Time,Type,Event,Value,Detail,Detail2
5937,stderr,received,'Traceback (most recent call last):'
5937,stderr,received,'  File "C:\\Users\\ali_a\\Desktop\\Single_Word_Processing_Stage\\Single_Word_Processing\\Christophe_Stimulation\\swp_localizer.py", line 50, in <module>'
5938,stderr,received,'    runs["image"].append(stimulus)'
5939,stderr,received,'    ~~~~^^^^^^^^^'
5940,stderr,received,"KeyError: 'image'"
